purchase_archive/__manifest__.py

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Please use the OCA manifest, https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py as template